hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/KeyDeletingService.java

```diff
@@ -348,14 +436,85 @@ private void processSnapshotDeepClean(int delCount)
               if (previousSnapshot != null) {
                 rcPrevOmSnapshot.close();
               }
+              if (previousToPrevSnapshot != null) {
```

Comment:
nit: you can use [IOUtils#closeQuietly()](https://github.com/apache/ozone/blob/bd110a477e5fc339aa3117f8facae06a1d2c7223/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/IOUtils.java#L91) to avoid null checks.
e.g. 
```
IOUtils.closeQuietly(rcPrevOmSnapshot, rcPrevToPrevOmSnapshot);
```